### ISSUE TITLE
Consider reorganizing sura names

### ISSUE THREAD

**Issue by ahmedre** [quran/quran_android#800]
Brother Abdul-Raqeeb, who added the German sura name translations, suggested something like this:

![media-20170204](https://cloud.githubusercontent.com/assets/47771/22624216/736a12e0-eb29-11e6-8e61-4a32474966da.jpg)

thoughts? cc @ozbek

**Comment by ozbek** [comment]
Meaning, _remove Makki/Madani and number of ayahs from subtitle,  and put surah translation instead_?

**Comment by ahmedre** [comment]
ah, good point - also the number of verses - i wonder if there's a way to reconcile all of those together. the problem with the German names especially is that they're way too long, so it looks pretty bad to have them next to the Arabic sura name.

**Comment by ozbek** [comment]
Not directly related, but German translation has repetitions (e.g., Ta-Ha is used both in original name and in translation): they could skip the translation if it is same as the original. Also, too many parentheses are used: using comma would be OK and would look better.

**Comment by Omerturger** [comment]
Or maybe like this?
![media-20170207](https://cloud.githubusercontent.com/assets/25547409/22677343/27478184-ecf2-11e6-9339-b822b77e43bc.png)

**Comment by ozbek** [comment]
@Omerturger 
Yes, I did think about that version. However, that will make list items of various height (as can clearly be seen from the screenshot you have posted) because not all surah names have translation.
